app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is intended to track all living animals. However, there is no mechanism to remove an animal from this list when it is no longer 'alive' (i.e., when its health is 0 or below). Consider adding logic to handle this in the `Animal` class or in the `Carnivore` class when an animal's health reaches 0.